src/homekit.cs

```diff
@@ -714,6 +735,10 @@ partial interface HMHomeDelegate {
 		[Export ("homeDidUpdateName:")]
 		void DidUpdateNameForHome (HMHome home);
 
+		[Watch (4,0), TV (11,0), iOS (11,0)]
+		[Export ("homeDidUpdateAccessControlForCurrentUser:")]
+		void DidUpdateAccessControl (HMHome home);
```

Comment:
I guess you're right, I understood it as the "Home" is the user or represents the user if I give it a bit more thoughts I don't think it's actually the case.

> @brief Informs the delegate when the access control for current user has been updated.